developer/src/kmc/test/generateKmnKeyboard.tests.ts

```diff
@@ -6,20 +6,21 @@ import 'mocha';
 import { assert } from 'chai';
 
 import { TestCompilerCallbacks } from '@keymanapp/developer-test-helpers';
-import { KeymanKeyboardGenerator } from '@keymanapp/kmc-generate';
+import { GeneratorOptions, KeymanKeyboardGenerator } from '@keymanapp/kmc-generate';
 
 import { unitTestEndpoints } from '../src/commands/generate.js';
-import { InfrastructureMessages } from '../src/messages/infrastructureMessages.js';
+import { KeymanTargets } from '@keymanapp/common-types';
 
 
 describe('generateKeymanKeyboard', function() {
-  const keyboardOptions = {
-    icon: true,
+  const keyboardOptions: GeneratorOptions = {
+    // icon: true,
```

Comment:
Was this supposed to be part of #12971?

Same change on the other tests.ts

Reply:
Yes, it was missed because it was essentially ignored (because it was an untyped object). Once I added the `GeneratorOptions` type, that forced the removal of the `icon` property. No real harm done, fortunately!